### ISSUE TITLE
Find templates automatically under .asciidoctor/templates

### ISSUE THREAD

**Issue by lask79** [asciidoctor/asciidoctor-vscode#843]
It currently is possible to add your own template files using `asciidoc.preview.templates`.
But the path has to be abolute.

Given that the extension already finds extensions under .asciidoctor/lib it would also be great if it also pick up templates under .asciidoctor/templates

**Comment by ggrossetie** [comment]
I think that's a good idea 👍🏻

**Comment by ggrossetie** [comment]
@ahus1 Do you think it makes sense to automatically load templates from `.asciidoctor/templates`?

**Comment by ahus1** [comment]
Makes sense to me, I like it. Do you have an example repository somewhere to test it?

Btw, would those templates work the same way for JRuby and the JS variants of Asciidoctor?

**Comment by ahus1** [comment]
It seems to make sense for a Asciidoctor.js setup, so let's proceed. I'll then see what I need to do to make it work with AsciidoctorJ one way or another.
